setup.py

```diff
@@ -22,6 +22,7 @@
     "pandas",
     "pyarrow",
     "pyrsistent",
+    "geopandas",
```

Comment:
I don't think this needs to be a required dependency. Let's handle this the way we handle Dask, etc. (That means we'll want to add an entry to the "Extra Packages" section of the "Get Started" docs.)

Reply:
Ah I see how protocols really work. The key is `BaseProtocol.value_is_valid()` method where it catches the exception if the optional dep is not satisfied. 

What you suggested makes sense, I'll change it accordingly.